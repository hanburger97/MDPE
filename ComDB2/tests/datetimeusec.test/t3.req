set timezone UTC
select id, dt, ds from dtus order by id
