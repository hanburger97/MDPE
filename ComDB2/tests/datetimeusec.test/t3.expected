(id=1, dt="1970-01-01T010000.246 UTC", ds=NULL)
(id=2, dt="1970-01-01T010000.246 UTC", ds=NULL)
(id=3, dt="1970-01-01T020000.246 UTC", ds=NULL)
(id=4, dt="2015-02-23T080000.246 UTC", ds=NULL)
(id=5, dt="2015-02-23T080000.000 UTC", ds=NULL)
(id=6, dt="2015-02-23T080000.246 UTC", ds=NULL)
(id=7, dt=NULL, ds="0 01:00:00.246")
(id=8, dt=NULL, ds="0 01:00:00.246")
(id=9, dt=NULL, ds="0 02:00:00.246")
(id=10, dt=NULL, ds="0 00:00:01.246")
(id=11, dt=NULL, ds="0 00:00:01.246")
(id=12, dt=NULL, ds="0 00:00:10.000")
(id=13, dt=NULL, ds="10 00:00:00.123")
(id=14, dt=NULL, ds="0 02:04:06.246")
